conformance/echo-basic/echo-basic.go

```diff
@@ -152,6 +161,38 @@ func delayResponse(request *http.Request) error {
 	return nil
 }
 
+func runH2CServer(h2cPort string, errchan chan<- error) {
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.ProtoMajor != 2 {
+			w.WriteHeader(http.StatusBadRequest)
+			fmt.Fprint(w, "Expected h2(c) request")
+			return
+		}
+
+		echoHandler(w, r)
+	})
+	h1s := &http.Server{
```

Comment:
Don't you mean `h2s` here?

Reply:
so there's an http2 package in x/net that has a `Server` - so it's technically not an h2c server

Reply:
I'll just call it h2c server